spec/services/claims/fetch_eligible_advocate_categories_spec.rb

```diff
@@ -1,66 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe Claims::FetchEligibleAdvocateCategories, type: :service do
+  before do
+    FeeScheme.find_by(name: 'LGFS', version: 9) || create(:fee_scheme, :lgfs_nine)
```

Comment:
**[not a blocker]** I've spoken offline with @colinbruce about this pattern and the more I see it, the more it feels that this should be centralised somewhere so that we avoid replicating the creation of all the fee schemes that should exist in the system in any spec we need them.

💭 Consider having this (at least) in some sort of helper in `spec/support` that we can call in the specs, so that if we decide to add/remove fee schemes, we can just change the helper code and leave the specs unchanged.